Resources/Private/Language/Tca.xlf

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
Do we also need the `encoding` and `standalone` attributes here?

Reply:
XML default encoding is UTF-8, I'm not sure if we would need a standalone attribute.